SKILL.tsv

```diff
@@ -2585,7 +2585,7 @@ SKILL_20150317_002584	{memo X}$Increases the physical attack of [Kneeling Shot]
 SKILL_20150317_002585	$Steady Aim: Enhance
 SKILL_20150317_002586	{memo X}$Increases the additional damage of [Steady Aim] per attribute level.
 SKILL_20150317_002587	$Sacrament: Enhance
-SKILL_20150317_002588	$Increases the damage on Holy property of [Sacrament] per attribute level.
+SKILL_20150317_002588	$Increases the Holy property damage of [Sacrament].
```

Comment:
Even though what Ragunaga said is right, I believe this is a textual error. This is an attribute skill, so it does change per attribute level. So it's best to just leave the 'per attribute level' in the line.